tools/decode-packets/src/main/scala/DecodePackets.scala

```diff
@@ -0,0 +1,165 @@
+import java.io.{File, ByteArrayOutputStream, FileOutputStream}
+import java.nio.charset.CodingErrorAction
+import net.psforever.packet.PacketCoding
+import scodec.bits._
+import scodec.Attempt.{Failure, Successful}
+import java.nio.file.{Files, Paths, StandardCopyOption}
+import scala.io.Codec
+import util.control.Breaks._
+import scala.collection.parallel.CollectionConverters._
+import scopt.OParser
+import sys.process._
+
+case class Config(
+    outDir: String = System.getProperty("user.dir"),
+    files: Seq[File] = Seq()
+)
+
+object DecodePackets {
+  def main(args: Array[String]): Unit = {
+
+    val builder = OParser.builder[Config]
+
+    val parser = {
+      import builder._
+      OParser.sequence(
+        programName("psf-decode-packets"),
+        opt[String]('o', "out-dir")
+          .action((x, c) => c.copy(outDir = x))
+          .text("Output directory"),
+        arg[File]("<file>...")
+          .unbounded()
+          .required()
+          .action((x, c) => c.copy(files = c.files :+ x))
+      )
+    }
+
+    val opts = OParser.parse(parser, args, Config()) match {
+      case Some(config) =>
+        config
+      case _ =>
+        sys.exit(1)
+    }
+
+    val outDir = new File(opts.outDir);
+    if(!outDir.exists()) {
+      outDir.mkdirs()
+    } else if(outDir.isFile()) {
+      println(s"error: out-dir is file")
+      sys.exit(1)
+    }
+
+     opts.files.foreach { file =>
+      if(!file.exists) {
+        println(s"file ${file.getAbsolutePath()} does not exist")
+        sys.exit(1)
+      }
+    }
+
+    opts.files.par.foreach { file =>
+      val parsedFile = s"gcapy -xa ${file.getAbsolutePath()}" !!
```

Comment:
The option to pass in pre-processed gcapy files would be nice addition. This actually wouldn't run on my machine because I run gcapy through a Linux instance using WSL, but do all my other dev through Windows.

Reply:
Yea I'll add an option